docs/whats_new/version_ongoing.rst

```diff
@@ -1,2 +1,19 @@
 Version ongoing
 ---------------
+
+Internal creation of proxy auxiliaries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is no longer necessary to manually defined a ``ProxyAuxiliary`` with
+``CCProxy``s yourself. If you simply pass the communication channel to
+each auxiliary that has to share it, ``pykiso`` will do the rest for you.
+
+
+Remove reduncancy of activate_log configuration parameter
```

Comment:
```suggestion
Remove redundancy of activate_log configuration parameter
```

Reply:
To me it is sufficient to have this behaviour explained with text and then illustrated with the schema linked below, adding a code snippet here will add a lot of content for a side-note the user shouldn't usually have to care about